tendrl/ceph_integration/sds_sync/__init__.py

```diff
@@ -122,7 +122,9 @@ def _run(self):
                     )
 
         while not self._complete.is_set():
-            gevent.sleep(10)
+            gevent.sleep(
```

Comment:
Should I grep for ttl and whenever there is ttl being set for the entities, change it to `sync_interval + 15` ?